src/status_im/android/core.cljs

```diff
@@ -169,7 +170,18 @@
                                        :contact-list-modal contact-list-modal)]
                        [component])]])]]))))})))
 
+(def exc (js/require "react-native-exception-handler"))
```

Comment:
The code related to `react-native-exception-handler` should be moved to separate ns and `register-exception-handler` needs to be added to `s.ios.core` ns.